src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book() {
+    }
+
+    public Book(String title, BigDecimal price) {
+        this.title = title;
+        this.price = price;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public BigDecimal getPrice() {
+        return price;
+    }
+
+    public void setPrice(BigDecimal price) {
+        this.price = price;
+    }
+
+    @Override
+    public String toString() {
+        return "Car("
+                + "id=" + id
+                + ", title=" + title
+                + ", price=" + price;
```

Comment:
There is a typo in the `toString` method. You're returning a string that starts with "Car(" instead of "Book(". This should be corrected to accurately represent the `Book` object.